<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QMainWindow" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>訂單管理</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="customer_name_input">
      <property name="placeholderText">
       <string>客戶名稱</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="item_input">
      <property name="placeholderText">
       <string>商品</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="quantity_input">
      <property name="placeholderText">
       <string>數量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="price_input">
      <property name="placeholderText">
       <string>價格</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_order_button">
      <property name="text">
       <string>新增訂單</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="generate_invoice_button">
      <property name="text">
       <string>生成發票</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>